examples/CommandBasedRobot/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -33,6 +33,7 @@
   private double yDirection;
   private double rotationSpeed;
```

Comment:
[nit] This should be rotationVelocity, since it can be positive or negative

Reply:
Is that true of both rotationSpeed and translationSpeed then? They should both be Velocity?

Reply:
Nope. `translationSpeed` really is just speed. It only specifies the magnitude of the motion (i.e. is always positive), and the direction of motion is given by `xDirection` and `yDirection`. `rotationSpeed` is really a velocity because it can be negative and includes both the magnitude and direction of motion